backend/api/publishing.py

```
@@ -305,33 +313,58 @@ async def get_scenario_full(
     reviews = db.query(ScenarioReview).options(
         selectinload(ScenarioReview.reviewer)
     ).filter(
         ScenarioReview.scenario_id == scenario_id
     ).order_by(desc(ScenarioReview.created_at)).limit(10).all()
-    
-    # Attach reviews to response
     scenario_dict = scenario.__dict__.copy()
     scenario_dict['reviews'] = reviews
-    
+    scenes = db.query(ScenarioScene).filter(ScenarioScene.scenario_id == scenario_id).order_by(ScenarioScene.scene_order).all()
+    from database.schemas import ScenarioSceneResponse
```

Ruff (0.12.2)

Redefinition of unused `ScenarioSceneResponse` from line 21

Remove definition: `ScenarioSceneResponse`

(F811)